lib/ctblperm.gi

```diff
@@ -213,16 +220,22 @@ local k,structures,ambiguousStructures,i,j,p,cem,ces,z,t,cen,a,
   fi;
 
   D.ids:=[];
+  D.chids:=[];
   D.rids:=[];
+  D.canreps:=[];
   for i in [1..D.klanz] do
     D.ids[i]:=D.identification(D,D.classreps[i]);
+    D.chids[i]:=D.cheapIdentification(D,D.classreps[i]);
     D.rids[i]:=
-     D.rationalidentification(D,D.classreps[i]);
+    D.rationalidentification(D,D.classreps[i]);
```

Comment:
Shouldn't the line be indented by one more space than before?

Reply:
Will indent (in original PR)